app/Repositories/Cts/MentorRepository.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Repositories\Cts;
+
+use App\Models\Cts\PositionValidation;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Support\Collection;
+
+class MentorRepository
+{
+    public function getMentorsWithin(int $rtsId): Collection
+    {
+        $positionValidations = PositionValidation::with(['member', 'position'])
+        ->whereHas('position', function (Builder $query) use ($rtsId) {
+            $query->where('rts_id', '=', $rtsId);
+        })->where('status', '=', 5)->get();
```

Comment:
What does the `5` mean?

Reply:
The status column being 5 is a signal (for CTS) that the user has the rights to mentor on that position.

Reply:
Could you add a variable to make it less magic?

Reply:
Hidden behind a scope - f3567b55

Reply:
Lovely stuff